sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/FileMetadataStructSuite.scala

```diff
@@ -654,4 +654,19 @@ class FileMetadataStructSuite extends QueryTest with SharedSparkSession {
       }
     }
   }
+
+  metadataColumnsTest("SPARK-41151: consistent _metadata nullability " +
+    "between analyzed and executed", schema) { (df, _, _) =>
```

Comment:
Indentation? We need two more spaces.

Reply:
Looks like the indentation is consistent at least in the test suite. Please check other test cases using metadataColumnsTest - they are all 2 spaces. The second line is not a parameter but a continuation of test name string.

Reply:
I'm curious why it's not a parameter? For me, the second line is a parameter because is **a part of the first parameter**. And, Apache Spark usually splits parameter definition sections from method definition sections, doesn't it?
> The second line is not a parameter but a continuation of test name string.

BTW, one thing I agree with @HeartSaVioR that we respect the nearest style in the code in general. So, I want to ask @Yaohua628 and @HeartSaVioR explicitly. Do you want to make this as an Apache Spark coding style officially? What I asking is that `the indentation on test case name splitting`.

Reply:
https://github.com/databricks/scala-style-guide#indent

For method declarations, use 4 space indentation for their parameters and put each in each line **when the parameters don't fit in two lines**. Return types can be either on the same line as the last parameter, or start a new line with 2 space indent.

So the rule of 4 space indentation is for 3+ lines of definition of method. We don't specifically mention about two lines of definition of method. While it's not super clear since we don't have strict guideline, but there is a general rule of spacing `Use 2-space indentation in general.`, which I think it could apply to cases which don't fall to exceptions.

Reply:
That's true. We have been unclear about this. Given that the general principle is to distinguish different sections for readability and we already use `2-space indentation` for *the method body*, I believe what we need is to extend the existing rule by removing `when the parameters don't fit in two lines`.

Mixing some part of the test case name and method body doesn't give us much readability. More worse, it's not extensible because that eventually leads us to use two-space indentation and four-space indentation in Case 1 and Case 2.
**Case 1**
```
metadataColumnsTest("SPARK-41151: consistent _metadata nullability " +
  "between analyzed and executed", schema) { (df, _, _) =>
```
**Case 2**
```
metadataColumnsTest("SPARK-41151: consistent _metadata nullability " +
    "long long long long long long long long long long long long long long " +
    "between analyzed and executed", schema) { (df, _, _) =>
```

Reply:
I was also incorrect. This is actually very clear:

https://github.com/databricks/scala-style-guide#indent

> For method and class constructor invocations, use 2 space indentation for its parameters and put each in each line when the parameters don't fit in two lines.

Technically, this is method "invocation", not method "definition".

Reply:
Ah, you are right. My bad.

Reply:
👍 Thanks for all the helpful discussions. Confirmed: 2-space indentation is consistent in this suite